commands/spinthewheel.js

```diff
@@ -4,14 +4,16 @@ let cooldowns = JSON.parse(fs.readFileSync("./pointsCooldowns.json", "utf8"));
 var WheelCooldowns = require('../models/WheelCooldown.js');
 exports.run = async(client, message, args, level) => { // eslint-disable-line no-unused-vars
     var today = Date.now();
-    var userCooldown = await client.GetWheelCooldown(message.author.id);
+    var userCooldown = new Date(await client.GetWheelCooldown(message.author.id));
+    console.log(userCooldown);
```

Comment:
Do we need this log here? If so, lets add some context about what is being logged.